src/examples/java/io/nats/examples/ExampleUtils.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2015-2018 The NATS Authors
```

Comment:
This can be debated, but for a new file, I usually put the year the file is created, so in this case 2018 only. If in 2019 you update it, it would then be 2018-2019, etc..